docs/userguide/datafiles.rst

```diff
@@ -364,10 +364,9 @@ Meanwhile, to further clarify the interplay between these three keywords,
 to include certain data file into the source distribution, the following
 logic condition has two be met::
 
-    (m or p) and not (i or e)  # TODO: why "not i"?
+    m or (p and not (i or e))  # TODO: why not i?
```

Comment:
@abravalheri Can you help me have a look here? why `not i` for source distribution here?

Reply:
> why `not i`?

Sorry, I copied over the wrong expression. This is the most updated https://github.com/abravalheri/experiment-setuptools-package-data?tab=readme-ov-file#results-for-setuptools6060.

```
s = m + e'.p
w = i.e'.m + e'.p  -- Which can be simplified => e' . (i.m + p)
                   -- and is also equivalent to => e' . (i.s + p)
```

`setuptools==58.3.0` had a but that made it operate in a non intuitive way.

Reply:
No problem, I noticed that! Thanks for the confirmation!